src/lib/extensions/dashboard/DashboardContextObject.ts

```diff
@@ -13,7 +15,7 @@ export function isDashboardContextObject(
   return (
     isContextObject(context) &&
     (context as DashboardContextObject).category === 'DASHBOARD' &&
-    (context as DashboardContextObject).hubId !== undefined &&
-    (context as DashboardContextObject).locationHref !== undefined
+    (context as DashboardContextObject).locationHref !== undefined &&
+    (context as DashboardContextObject).hub !== undefined
```

Comment:
`hubId` is deprecated but not removed so should probably still consider it to be a dashboard context even if only hubId set?